### ISSUE TITLE
Feature Request: Ability to Recharge

### ISSUE THREAD

**Issue by BillyFKidney** [barleybobs/homeassistant-ecowater-softener#37]
Would it be possible to add the ability to Recharge Now or Schedule Recharge?

<img width="343" alt="image" src="https://github.com/barleybobs/homeassistant-ecowater-softener/assets/65253277/50e2335c-6cbc-474d-bbbe-d6060246a45d">

**Comment by niallobr** [comment]
Just to add if this was ever considered, it would be good to see how much time is remaining during a recharge.

I just cleaned out my unit during the daytime, refilled and started a recharge, so it would be useful for me and my family to know when this is finished.

This time remaining is displayed on the screen on my Compact 200, not sure if it gets exposed to the API.

It could be good if the ‘Valve Position’ contained in the EASE report was exposed also to see what stage things are at e.g. “brine_slow_rinse”.

**Comment by Lk1975** [comment]
Yes, it would be really nice to have, also to be able to do some out-of-schedule recharge, i.e. using data from other sensors to start an automation.
